fix: remove RENALCL parameters from Parameter Selection (wrongly calculated in PKNCA)

Gero1999 opened the pull request:
## Issue

Closes #780 

## Description
We remove RENALCL parameters from `metadata_nca_parameters` as they are wrongly calculated in PKNCA

- RENALCL is no longer calcualted through PKNCA, so it is not available in `Parameter Selection`. 
- Parameter Ratios will recognise it if introduced

## Definition of Done
- [x] All RENALCL parameters do not appear in the parameters input

## How to test
`Urine dataset > Select Urine PCSPEC > In parameter selection no RENALCL is available`
In addition, compute it through parameter ratios:

`TestGroups = PCSPEC: URINE`, `TestParameter = RCAMINT`, `RefParameter = AUCLST/AUCIFO/AUCIFP`

Remember to select multiple doses (ATPTREF) so there are results available for comparison

Make sure the label appears as well in the results


## Contributor checklist
- [ ] Code passes lintr checks
- [ ] Code passes all unit tests
- [ ] New logic covered by unit tests
- [ ] New logic is documented
- [ ] App or package changes are reflected in NEWS
- [ ] Package version is incremented

Gero1999 committed:
fix: add new definitions for clr parameters (RENALCL)

Gero1999 committed:
news: add as a temporary fix in case is auto-solved in the future

Gero1999 committed:
spelling: update wordlist

Gero1999 committed:
merge: solving easy conflicts in NEWS.md & WORDLIST

Merge remote-tracking branch 'origin/main' into 780-bug/renalcl-only-calculated-if-rcamint

# Conflicts:
#	NEWS.md
#	inst/WORDLIST

Gero1999 committed:
data: rm 3 RENALCL parameters from metadata_nca_parameters

Gero1999 committed:
rm adjustments done for clr parameters (no longer calculated)

Gero1999 committed:
news: inform user of RENALCL removal

Gero1999 committed:
merge: solve conflicts metadata_nca_parameters & add 1 row RENALCL

Merge remote-tracking branch 'origin/main' into 780-bug/renalcl-only-calculated-if-rcamint

# Conflicts:
#	NEWS.md
#	data-raw/metadata_nca_parameters.csv
#	data/metadata_nca_parameters.rda

Gero1999 committed:
feat: recognise in the ratio table RENALCL

js3110 commented on inst/shiny/modules/tab_nca/setup/ratio_calculations_table.R:
```diff
@@ -311,6 +311,9 @@ ratios_table_server <- function(
         startsWith(RefGroups, profile_col) ~ paste0("AR", TestParameter),
+        startsWith(TestGroups, paste0(pcspec_col, ": URINE")) &
+          startsWith(TestParameter, "RCAMINT") &
+          startsWith(RefParameter, "AUC") ~ "RENALCL",
```
```suggestion
        startsWith(TestGroups, paste0(pcspec_col, ": URINE")) &
          startsWith(TestParameter, "RCAMINT") &
          startsWith(RefParameter, "AUC") ~ "RENALCL",
        startsWith(RefGroups, analyte_col) ~ paste0("MR", TestParameter),
        startsWith(RefGroups, profile_col) ~ paste0("AR", TestParameter),
```

js3110 reviewed (commented):
If i rty to create RENALCL via the ratios it doesn't automatically call it RENALCL:

<img width="1343" height="157" alt="image" src="https://github.com/user-attachments/assets/fbe52c30-c698-4c86-9841-c559d2a9d8e6" />


Unless I change the Ref group to PCSPEC: URINE
<img width="1176" height="102" alt="image" src="https://github.com/user-attachments/assets/ec689f59-482f-4481-a81b-91e4dc77e8ab" />


Which obvs isnt correct. From the code I think this part needs to be specified above the others in the `case_when`? as otherwise the `AR, TestParameter` part takes priority

Gero1999 commented:
Hmm let me check I don't think the problem is there for what you described

Gero1999 committed:
fix: flexibilize reactivity & increase accuracy in denomination

Gero1999 commented:
Should be fine now :)

js3110 reviewed (approved):
Awesome Sauce!!!

Gero1999 committed:
news: correct English phrasing

Gero1999 committed:
bump pkg version 0.1.0.9020

Gero1999 committed:
Merge branch 'main' into 780-bug/renalcl-only-calculated-if-rcamint

Gero1999 committed:
merge: solve conflicts of pkg version & add metadata ER rows

Merge remote-tracking branch 'origin/main' into 780-bug/renalcl-only-calculated-if-rcamint

# Conflicts:
#	DESCRIPTION
#	data-raw/metadata_nca_parameters.csv
#	data/metadata_nca_parameters.rda
